frontend/src/pages/LandlordPage.tsx

```diff
@@ -8,12 +8,16 @@ import Review from '../components/Review/Review';
 import ReviewHeader from '../components/Review/ReviewHeader';
 import AppBar, { NavbarButton } from '../components/utils/NavBar';
 import { useTitle } from '../utils';
+import LandlordHeader from '../components/Landlord/Header';
 
 type LandlordData = {
   properties: string[];
   photos: string[];
   phone: string;
   address: string;
+  name: string;
+  overallRating: number;
+  numReviews: number;
```

Comment:
There's a Landlord type in common/types/db-types, which includes the name and overallRating fields and has a reviews field which is an array you can get the length of instead of adding the field numReviews, so maybe use that Landlord type instead of LandlordData